daemon/options/bpf.go

```diff
@@ -32,13 +32,15 @@ type BpfConfig struct {
 	Cgroup2Path      string
 	EnableMda        bool
 	BpfVerifyLogSize int
+	EnableBpfLog     bool
 }
 
 func (c *BpfConfig) AttachFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&c.BpfFsPath, "bpf-fs-path", "/sys/fs/bpf", "bpf fs path")
 	cmd.PersistentFlags().StringVar(&c.Cgroup2Path, "cgroup2-path", "/mnt/kmesh_cgroup2", "cgroup2 path")
 	cmd.PersistentFlags().StringVar(&c.Mode, "mode", "workload", "controller plane mode, valid values are [ads, workload]")
 	cmd.PersistentFlags().BoolVar(&c.EnableMda, "enable-mda", false, "enable mda")
+	cmd.PersistentFlags().BoolVar(&c.EnableBpfLog, "enable-bpf-log", false, "enable ebpf log in daemon process")
```

Comment:
Is the bpf log function turned off by default?

Reply:
I am also not sure whether need to enable ebpf log in userspace , I wrote this code just as yesterday said, it is like a feature gateway.

Reply:
> I am also not sure whether need to enable ebpf log default  in userspace , I wrote this code just as yesterday said, it is like a feature gateway.

Reply:
> Good job~ The following two points need to be confirmed to determine whether tunned off by default:
> 
> 1. Can the 5.10 kernel version run properly?
> 2. Test the interface performance.

for 2, possiblely it won't turn on in product env, I think it only need to run this feature when user want to debug or development Kmesh. BTW, how to test this kind of interface?

Reply:
Log needs to be opened in the production environment, otherwise there is no way to locate the problem.
We can write a test example, generate a batch of logs and count the time spent.

Reply:
In the production environment, we can use the log level to control the printing of only critical error messages to avoid massive log output.

Reply:
For 1, I had searched 5.10 kernel code, the `bpf_snprintf` or `BPF_SNPRINTF` not exist.